src/components/tables/AccessGrants/AccessLinks/Dialog/GenerateInvitation.tsx

```diff
@@ -63,6 +134,7 @@ function GenerateInvitation({
         generateInvitation: (event: React.MouseEvent<HTMLElement>) => {
             event.preventDefault();
 
+            const objectRole = prefix + name;
```

Comment:
We prefer string templates... which I am actually surprised isn't an eslint rule already.

This should not always add the `name`
<img width="1067" height="846" alt="Image" src="https://github.com/user-attachments/assets/84d7e887-4497-4f6a-8c6e-7a316dcaff78" />